deploy-prod.sh

```diff
@@ -1,3 +1,5 @@
+rm -rf ./build
```

Comment:
is there any reason to remove it? Its always good to start with a clear environment. What ifg this script is not run on CI?